### PR TITLE
ID: 0000-Add CSV export for temperature readout

### PR DESCRIPTION
**Pull request by nuria-balaguer**

### CODE REVIEW

File: src/Button/Button.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
+/* eslint-disable complexity */
```
Comment (ManilnikovaNastiia) [inline src/Button/Button.js:1]: it looks like there is no problem with complexity, could you try without eslint-disable

File: src/TemperatureRange/TemperatureRange.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+import useStyles from './TemperatureRange.styles'
+
+const propTypes = {
+    showTempRange: PropTypes.bool.isRequired,
+    onCheckShowTempRange: PropTypes.func.isRequired,
+    title: PropTypes.string.isRequired,
+}
+
+const defaultProps = {}
```
Comment (ManilnikovaNastiia) [inline src/TemperatureRange/TemperatureRange.js:14]: remove it

File: src/TemperatureRange/TemperatureRange.styles.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import {
```
Comment (ManilnikovaNastiia) [inline src/TemperatureRange/TemperatureRange.styles.js:1]: as I remember on last retro we decided to not create empty style file

**Review by ManilnikovaNastiia — APPROVED** [review]

**Review by ricardovoegeli — APPROVED** [review]

File: src/Button/Button.js (hunk 1 above)
Comment (nuria-balaguer) [reply src/Button/Button.js]: fixed

File: src/TemperatureRange/TemperatureRange.styles.js (hunk 3 above)
Comment (nuria-balaguer) [reply src/TemperatureRange/TemperatureRange.styles.js]: fixed

**Review by ricardovoegeli — DISMISSED** [review]

File: src/TemperatureRange/TemperatureRange.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+const TemperatureRange = (props) => {
+    const {
+        showTempRange,
+        onCheckShowTempRange,
+        title,
+    } = props
+
+    return (
+        <div>
```
[the remark this answers is not in the document]
Comment (nuria-balaguer) [reply src/TemperatureRange/TemperatureRange.js]: fixed

**Review by ricardovoegeli — APPROVED** [review]

**Review by ricardovoegeli — DISMISSED** [review]

**Review by ManilnikovaNastiia — DISMISSED** [review]

**Review by ManilnikovaNastiia — APPROVED** [review]

**Pull request merged**